api/configuration/health.ts

```diff
@@ -0,0 +1,20 @@
+import {EnvironmentConfigServices} from '../interfaces/environment.config'
+import {getConfigValue} from './index'
+import { SERVICES_CCD_COMPONENT_API_PATH } from './references'
```

Comment:
Why are these 5 imports all done separately?

Reply:
just matching the MO implementation